internal/config/image/cuda_image.go

```diff
@@ -271,6 +282,28 @@ func (i CUDA) DevicesFromMounts() []string {
 	return devices
 }
 
+func (i CUDA) GetDevices(acceptDeviceListAsVolumeMounts, acceptEnvvarUnprivileged bool) []string {
```

Comment:
```suggestion
func (i CUDA) VisibleDevices() []string {
```

Reply:
Possibly out of scope for this change, but the same logic needs to be implemented for IMEX channels and CDI devices.